cmd/restic/cmd_prune.go

```diff
@@ -419,6 +421,10 @@ func prune(opts PruneOptions, gopts GlobalOptions, repo restic.Repository, usedB
 		pi := repackCandidates[i].packInfo
 		pj := repackCandidates[j].packInfo
 		switch {
+		case opts.RepackSmall && pi.unusedSize+pi.usedSize < uint64(repo.MinPackSize()) && pj.unusedSize+pj.usedSize >= uint64(repo.MinPackSize()):
```

Comment:
These two cases should be moved last in the switch-statement. Cleaning up other things is probably more important than getting rid of a few too small files.